modules/ml/src/main/java/org/apache/ignite/ml/genetic/MutateTask.java

```diff
@@ -78,7 +78,7 @@ public MutateTask(GAConfiguration cfg) {
      * @param nodes List of ClusterNode
      * @param chromosomeKeys Primary keys for respective chromosomes
```

Comment:
Where is the return statement in javadoc?

Reply:
fixed